renovate.json

```diff
@@ -440,7 +446,7 @@
       "description": "Update Wolfi base image",
       "customType": "regex",
       "fileMatch": [
-        "^src/dev/build/tasks/os_packages/docker_generator/run\\.ts$"
+        "\\.ts$"
```

Comment:
This is going to search 43k files, double that if it matches node_modules.  This seems expensive..are we concerned about performance?

Reply:
It's non-negligible, so I think reverting it is probably best, just to be safe?

Reply:
The source file hasn't changed location since inception, and a comment in the file noting renovate.json is close enough for me.  In the interim, I'm getting notifications elsewhere and we can look into a pre-publish pre-publish that validates there isn't a new version.

Reply:
Sounds good to me, was just responding to the comments around brittleness in slack.